platform.txt

```diff
@@ -1,5 +1,5 @@
-name=ESP32 Arduino
-version=0.0.1
+name=ESP32 Arduino 1.0.1 DEV
```

Comment:
revert this line please :)

Reply:
But, the version number by itself does not show up anywhere. Without it, this mod is useless.